pyproject.toml

```diff
@@ -172,3 +172,28 @@ style = "pep440"
 versionfile_source = "xdsl/_version.py"
 versionfile_build = "xdsl/_version.py"
 tag_prefix = "v"
+
+[tool.coverage.run]
+branch = true
+omit = [
+    # Autogenerated missed code handles other VCSes.
+    "xdsl/_version.py",
+    "tests/filecheck/frontend/*",
+]
+concurrency = ["multiprocessing"]
+parallel = true
+source = ["xdsl/", "tests/", "docs/"]
```

Comment:
didn't we just remove `tests/` in the other PR?

Reply:
Indeed, I've just merged that one, will remove tests from here before I merge